### ISSUE TITLE
Scroll up/down the page programmatically

### ISSUE THREAD

**Issue by nearsyh** [spolu/breach_core#192]
I guess it is not possible with current api. I'm not sure if this is common, what I want to do is making a vimmode plugin for breach, which is similar to [Vimium](https://chrome.google.com/webstore/detail/vimium/dbepggeogbaibhgnhhndojpepiihcmeb?hl=en) for chrome.

**Comment by kyleshrader** [comment]
Unrelated to breach, check out vimperator for firefox :D It's amazing.

**Comment by spolu** [comment]
Yep such a module for breach would be really really cool. We could go as far as not have any tab system visible when running it... Just a small shell at the bottom maybe!

**Comment by nearsyh** [comment]
Yep. But I still wonder if scrolling is possible.

**Comment by kyleshrader** [comment]
I am working on a vimperator style clone; however, idk how far I'll get, or how well I'll do in comparison to vimperator. :P http://www.github.com/kylekinkade/mod_vimline

**Comment by spolu** [comment]
Let me now how I can help!

**Comment by spolu** [comment]
And make sure to add your project there: https://github.com/breach/breach_core/wiki/List-of-modules

**Comment by spolu** [comment]
So scrolling, would probably not be possible at the moment. We would need to add an API to the ExoBrowser or wait for the next version...

**Comment by spolu** [comment]
Referencing this issue to track progress: https://github.com/breach/exo_browser/issues/161
